devlarabar/100freelancers
# Optimize filtering clients by business type

### Description

- Previous filtering implementation required the client array to be duplicated into 2 state variables and required manually filtering the client list. Current implementation requires only one state variable and automatically filters the client list.
- Bugfix: Removes duplicate business types being displayed in the filter list.

### Testing

- Duplicate business type bugfix test
  1. Create 2 clients with the same business type
  2. Click on the filter icon
  3. Assert that the business type is shown once in the dropdown.

### Did you change the schema?
